install/kubernetes/cilium/values.yaml

```diff
@@ -510,6 +510,36 @@ daemon:
   # -- Configure where Cilium runtime state should be stored.
   runPath: "/var/run/cilium"
 
+  # -- Configure a custom list of possible configuration override sources
+  # The default is "config-map:cilium-config,cilium-node-config,node"
+  # Possible entries are
+  #   - "configmap:<namespace>/<name>" -- a ConfigMap with name <name>, optionally at a namespace.
+  #   - "cilium-node-config:<namespace>/<name>" --  any or all CiliumNodeConfigurations. If <name> is specified,
+  #                                                 then only that one is considered. Otherwise, all CNCs in the given namespace
+  #                                                 that match the LabelSelector are used
+  #   - "node" -- Any annotations on the Node object of the form "io.cilium.config/<KEY> = <VALUE>"
+  #
+  # Note that this value should be a comma-separated string.
+  configSources: ~
+
+  # -- allowedConfigOverrides is a list of config-map keys that can be overridden.
+  # That is to say, if this value is set, config sources (excepting the first one) can
+  # only override keys in this list.
+  #
+  # This takes precedence over blockedConfigOverrides.
+  #
+  # By default, all keys may be overridden. To disable overrides, set this to an invalid key.
+  allowedConfigOverrides: ~
+
+  # -- blockedConfigOverrides is a list of config-map keys that may not be overridden.
+  # In other words, if any of these keys appear in a configuration source excepting the
+  # first one, they will be ignored
+  #
+  # This is ignored if allowedConfigOverrides is set.
+  #
+  # By default, all keys may be overridden.
+  blockedConfigOverrides: ~
```

Comment:
from a user-perspective, an `any` key like `blockedConfigOverrides: *` might be nice vs. setting an invalid key for `allowedConfigOverrides` to block overrides

Reply:
very good point; I'll add that sugar.

Reply:
I added "allowedConfigOverrides: none" (which isn't actually a code change), and at Joe's suggestion, also suggested disabling overrides above.